forge-gui/res/cardsfolder/l/loose_in_the_park.txt

```diff
@@ -0,0 +1,13 @@
+Name:Loose in the Park
+ManaCost:1 G
+Types:Enchantment Aura
+K:Enchant land
+A:SP$ Attach | ValidTgts$ Land | AILogic$ Pump
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ FreeCard | TriggerDescription$ When CARDNAME enters the battlefield, draw a card, then draft a card from CARDNAME's spellbook and exile it.
+SVar:FreeCard:DB$ Draw | Defined$ You | NumCards$ 1 | SubAbility$ TrigDraft
+SVar:TrigDraft:DB$ Draft | TriggerZones$ Battlefield | Spellbook$ Bristling Boar,Enraged Ceratok,Exuberant Wolfbear,Gaea's Protector,Master Symmetrist,Ornery Dilophosaur,Overgrown Armasaur,Predatory Wurm,Prized Unicorn,Sabertooth Mauler,Spike-Tailed Ceratops,Spore Crawler,Thrashing Brontodon,Wardscale Crocodile,World Shaper | Zone$ Exile | RememberDrafted$ True | TriggerDescription$ Draft a card from CARDNAME's spellbook and exile it.  
```

Comment:
You can just get rid of the TriggerDescription$ in L8 now, doesn't to anything

Thanks :)